keras_nlp/models/llama/llama_layernorm.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from keras_nlp.backend import keras
+from keras_nlp.backend import ops
+
+
+class LlamaLayerNorm(keras.layers.Layer):
```

Comment:
This looks like it is just layernorm with `center=False`. Is that true? Can we just use that?

Reply:
We cannot, see https://github.com/keras-team/keras-nlp/issues/1207